### PR TITLE
[#566] Add versioning for kubernetes infrastructure

### PR DESCRIPTION
**Pull request by serglupach**

### CODE REVIEW

File: deploy/helms/oauth2-proxy/values.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@
 replicaCount: 1
 
 image:
-  repository: docker.io/sath89/oauth2_proxy
-  tag: kubernetes
+  repository: legionplatform/k8s-oauth2-proxy-docker
```
Comment (alexander-semenets) [inline deploy/helms/oauth2-proxy/values.yaml]: do we really need "-docker" suffix for this image?

Comment (serglupach) [reply deploy/helms/oauth2-proxy/values.yaml]: Not needed. Removed suffix.

**Comment by alexander-semenets** [comment]
bot, test all

**Comment by serglupach** [comment]
bot, test all

**Comment by serglupach** [comment]
bot, test all

**Comment by serglupach** [comment]
bot, test all

**Pull request merged**